### PR TITLE
Ensure DNS requests are sent down the VPN on Ubuntu

### LINKED ISSUE #508: Ubuntu Network Manager instructions don't send DNS queries down VPN
Hi,

The default instructions provided to configure Network Manager on Ubuntu don't include the instructions to make use of the DNS server down the VPN tunnel. This could leak DNS lookups onto the local network, depending on resolver configuration. I believe the following should be added to the manual instructions already present while editing the VPN connection but before going into the advanced screen:

* Go to the 'IPv4 Settings' tab
* Change 'Method' to 'Automatic (VPN) addresses only'
* Add _[IP address of OpenVPN tunnel on server side, e.g. 10.9.0.1]_ into the 'DNS Servers' box.
* Go back to the VPN tab and click 'Advanced'...
* etc

### PR DESCRIPTION
**Pull request by Xiol**
Fixes #508.

### CODE REVIEW

**Comment by sjug** [comment]
Thanks @Xiol

File: playbooks/roles/openvpn/templates/instructions.md.j2 [hunk 1]
```diff
@@ -102,7 +102,10 @@ It's preferable to configure Ubuntu using the OpenVPN plugin for NetworkManager.
 1. Select `client.crt` from the client-files directory of your choice for the *User Certificate*.
 1. Select `ca.crt` from the same client-files directory for the *CA Certificate*.
 1. Select `client.key` from the same client-files directory for the *Private Key*.
-1. Click the *Advanced* button.
+1. Go to the *IPv4 Settings* tab
+   * Change the *Method* to *Automatic (VPN) addresses only*
+   * Add `{{ openvpn_server }}` into the DNS Servers box.
```
Comment (jlund) [inline playbooks/roles/openvpn/templates/instructions.md.j2]: I think we'd want to use `{{ dnsmasq_openvpn_tcp_ip }}` here instead.

**Comment by cpu** [comment]
@Xiol Are you interested in addressing the feedback @jlund left on Apr 5th? Alternatively would you like someone to take over or close this PR?

**Comment by cpu** [comment]
I'm going to close this PR. There hasn't been an answer to my question from May 21st. Please feel free to reopen if you're interested in addressing the feedback on the previous submission & can rebase against master. Thanks!

**Pull request closed without merging**